scanner/package-lock.json

```diff
@@ -0,0 +1,4387 @@
+{
```

Comment:
Please remove all `package-lock.json` files added by `npm`. We use `pnpm`. Also please add this `package-lock.json` to `.gitignore` so it will not get added again.